templates/ruby/lib/container/payload.rb.twig

```diff
@@ -0,0 +1,77 @@
+module Appwrite
+  class Payload
+    attr_reader :data
+    attr_reader :filename
+
+    def initialize(data: nil, path: nil, filename: nil)
+      @path = path
+      @data = data
+      @filename = filename
+
+      @size = if @data then
+        @data.bytesize
+      else
+        File.size(@path)
```

Comment:
Do we need this size?

Reply:
It is stored to enable loading the file in chunks